site/content/docs/main/restore-hooks.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Restore Hooks"
+layout: docs
+---
+
+Velero supports Restore Hooks, custom actions that can be executed during or after the restore process. There are two kinds of Restore Hooks:
+
+1. InitContainer RestoreHooks: These will add init containers into restored pods to perform any necessary setup before the application containers of the restored pod can start.
```

Comment:
I'd actually lean towards `InitContainer Restore Hooks` and `Exec Restore Hooks` when talking about the concepts (as opposed to the variables/annotations) in this short list and in the headings. I see this being the exact same thing being discussed in L21, which breaks it into 2 words, but I'm curious how others might read it.

I think this is similar to how we call our hooks on the backup side `Backup Hooks`, even though those don't have 2 subtypes.

That said, I'm willing to hear what others think on this, as getting terminology right is important.